frontends/web/src/api/bitbox02.ts

```diff
@@ -32,6 +32,10 @@ type DeviceInfoResponse = SuccessResponse & {
     deviceInfo: DeviceInfo;
 };
 
+export const resetDevice = (deviceID: string): Promise<SuccessResponse> => {
+  return apiPost(`devices/bitbox02/${deviceID}/reset`);
+};
```

Comment:
should return `Promise<SuccessResponse | FailResponse>` here is the backend part  
https://github.com/digitalbitbox/bitbox-wallet-app/blob/0eef103490f729ca7d73afa78a1dfbdeaa370a70/backend/devices/bitbox02/handlers/handlers.go#L303-L309

Reply:
Thanks! changes, made. PTAL 🙏